frontend/src/pages/talks/Talks.tsx

```diff
@@ -52,21 +52,27 @@ const Talks = () => {
 
   useEffect(() => {
     async function getTalksEntries() {
-      try {
-        const talks = TalksService.talksList();
-
-        if (!talks.results || talks.results?.length === 0) {
-          return;
-        }
+      const talks = await BackofficeService.backofficeTalksList({
+        limit: 100,
+      }).catch(() => {
+        contactAdministrator('error');
+      });
 
+      if (talks && talks.results) {
         const now = new Date();
 
         const sortedTalks = talks.results.reduce<{
           past: TalkEntry[];
           future: TalkEntry[];
         }>(
           (acc: SortedTalkAccumulator, talk: TalkEntry) => {
+            // Do not display not scheduled Talk.
+            if (!talk.date) {
+              return acc;
+            }
+
             const talkDate = new Date(talk.date);
+
             if (talkDate < now) {
               acc.past.push(talk);
             } else {
```

Comment:
We might want to keep some tolerance here to keep the invite link accessible even if the talk has just started. 

To move the talk to the "Past" section after 2 hours:
```suggestion
            if (talkDate < (new Date(now.getTime() - 2 * 3600 * 1000))) {
              acc.past.push(talk);
            } else {
```